From 2c267b01c627bc6f96e7de17b17563d0c07df829 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Mon, 26 Feb 2024 17:22:55 +0100
Subject: [PATCH 0714/1695] .gitignore: Change entries to only ignore in the
 source root folder
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Previously, each gitignore entry was ignored all across the project.
This patch allows to ignore only entries that are in the source root
folder, except for editor's swap files.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 32fc7da6717d0c0dad64aca0e0d0cc16c8fbb338 .gitignore: Change entries to only ignore in the source root folder]
Change-Id: I5ef2b08ab78670ada1a4f693fec95c2430210c10
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
(cherry picked from commit 5b4a782ebccfb447f3384b1def7e6406dea1dd9c)
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/371642
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 .gitignore | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/.gitignore b/.gitignore
index 57188550b..c7a0bbe38 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,8 +1,8 @@
-.cproject
-compile_commands.json
-cscope.*
-tags
-TAGS
-out
-.project
+/.cproject
+/compile_commands.json
+/cscope.*
+/tags
+/TAGS
+/out
+/.project
 *.swp
-- 
2.47.2

